### DISCUSSION TITLE
Text width

### DISCUSSION THREAD

**Discussion by panosGkrigkas** [Bodmer/TFT_eSPI#863] [Q&A - Fonts] [answered]
First of all thank you for your time and effort on this library! Magnificent work!
The following question might be stupid or have nothing to do with the library(let me know if I have to remove it or delete it)... Is there a way to predetermine the text length before sending it to print on display? I try to test with the same font size how much letters I can fit in a single line but it changes depending on different inputs. eg. I can have 23 "a" on a single line before dropping to the next one but I can fit 25 numbers...

**Comment by Bodmer** [comment, marked as answer]
[These functions](https://github.com/Bodmer/TFT_eSPI/blob/master/TFT_eSPI.h#L541-L546) return the width of a string or character in pixels.

**Reply by panosGkrigkas** [reply]
Should have searched more! Thank you!

**Reply by aa2mz** [reply]
Currently lines 680 to 683
